### PR TITLE
Add EIP: Non-fungible Token Bound Accounts

### PR DESCRIPTION
**Pull request by jaydenwindle**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-6551.md [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+Non-fungible tokens are increasingly becoming a form of on-chain identity. This follows quite naturally from the ERC-721 specification - each non-fungible token has a globally unique identifier, and by extension, a unique identity.
+
+Unlike other forms of on-chain identity, ERC-721 tokens cannot act as an agent or associate with other on-chain assets. This limitation stands in contrast with many real-world instances of non-fungible assets. For example:
+
+- A character in a role-playing game that accumulates assets and abilities over time based on actions they have taken
+- An automobile composed of many fungible and non-fungible components
+- An automated investment portfolio composed of multiple fungible assets
+- A punch pass membership card granting access to an establishment and recording a history of past interactions
+
+Several proposals have attempted to give ERC-721 tokens the ability to own assets. Some notable proposals include [ERC-998](./eip-998.md) and [ERC-6150](./eip-6150.md).
```
Comment (SamWilsn) [inline EIPS/eip-6551.md:31]: I would very much recommend removing these references. Your proposal will not be able to advance past `Draft` unless all proposals linked also advance past `Draft`.

```suggestion
Several proposals have attempted to give ERC-721 tokens the ability to own assets.
```

File: EIPS/eip-6551.md [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+ERC-1167 Footer               (15 bytes)
+STOP code                     (1 byte)
+<chainId (uint256)>           (32 bytes)
+<tokenContract (address)>     (32 bytes)
+<tokenId (uint256)>           (32 bytes)
+```
+
+For example, the token bound account with implementation address `0xbebebebebebebebebebebebebebebebebebebebe`, chain ID `1`, token contract `0xcfcfcfcfcfcfcfcfcfcfcfcfcfcfcfcfcfcfcfcf` and token ID `123` would have the following deployed bytecode:
+
+```!
```
Comment (SamWilsn) [inline EIPS/eip-6551.md:78]: I'm not familiar with the `!` notation. What is it supposed to do?

File: EIPS/eip-6551.md (hunk 1 above)
Comment (jaydenwindle) [reply EIPS/eip-6551.md:31]: Ah, didn't realize that. Will remove!

File: EIPS/eip-6551.md (hunk 2 above)
Comment (jaydenwindle) [reply EIPS/eip-6551.md:78]: Ooops, composed this in hackmd where the `!` turns on line wrapping. Will remove :)

**Review by SamWilsn — APPROVED** [review]

**Comment by Pandapip1** [comment]
@eth-bot rerun

**Comment by Pandapip1** [comment]
@eth-bot rerun

**Review by tuanggo — COMMENTED** [review]
open

**Review by Hawthorne001 — APPROVED** [review]

**Review by Elenore23 — APPROVED** [review]

**Review by hazemdr454 — COMMENTED** [review]
H

**Review by noelportillo — COMMENTED** [review]
The most important

**Review by ghost — APPROVED** [review]

**Review by Eusinho1985 — APPROVED** [review]

**Review by falrefaei-debug — APPROVED** [review]

**Review by Danyalkasiri — COMMENTED** [review]
###

**Pull request merged**